### DISCUSSION TITLE
🔗 Add Webp uploading support; Responsive option on Gradient Background; Responsive option on Box Shadow (Duplicate of #10247) - [ED-9310]

### DISCUSSION THREAD

**Discussion by sarmadali777** [elementor/elementor#15377] [Feature Request]
Respected Elementor team Thanks for the great product 🙌. Day by Day you people adding great features I request you to please add three more features. 
- [ ] Image Webp uploading support.
- [ ] Add Responsive option on gradient background.
- [ ] Add Responsive option or border shadow.

Thanks

**Comment by arsanet** [comment]
What do you mean by Add Webp uploading support? Right now, wherever there is a place for uploading images, you can also upload images in webp format. It is also an image, only the format is considered a newer format.
